server/messages.proto

```diff
@@ -23,4 +23,7 @@ message ReceiveRequest{
 }
 message ReceiveResponse{
     bytes payload = 1;
+}
+message PartyInfo{
+    bytes payload = 1;
```

Comment:
The `PartyInfo` message should most of the fields contained in the `PartyInfo` type, e.g.:
```
url string  // URL identifying this node
recipients map[[nacl.KeySize]byte]string  // public key -> URL
parties    map[string]bool  // Node (or party) URLs
```

Otherwise we're still relying on the encoding libraries in Crux instead of fully using protobuf.